insights/util/__init__.py

```diff
@@ -93,17 +93,15 @@ def __init__(self, dict_):
         self.__dict__ = dict_
 
 
-def parse_table(content, delim=None, max_splits=-1, strip=True):
+def parse_table(content, delim=None, max_splits=-1, strip=True, header_split=True):
     """
     Parses table-like text. Assumes the first row contains column names.
     Column names cannot contain spaces.  Fields cannot be blank.
     """
     if not content:
         return []
-    if strip:
-        cols = [c.strip() for c in content[0].split(delim)]
-    else:
-        cols = content[0].split(delim)
+    cols = [c.strip() if strip else c for c in content[0].split(delim if header_split else None)]
```

Comment:
@SteveHNH I went looking for the tests for this method and could not find any.  Would you mind to add tests?

Reply:
Sure! I'll get all the above changes put in and add a test as well.
Thanks, @bfahr